app/views/guides/show.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <% content_for :sidebar do %>
   <% if @journey.include?(@guide) %>
-    <div class="link-promo">
+    <div class="link-promo ga-journey-sidebar">
```

Comment:
I think data attributes are more appropriate than classes for targeting JS behaviour like this